CRM/Upgrade/Incremental/sql/4.7.17.mysql.tpl

```diff
@@ -7,3 +7,28 @@ UPDATE civicrm_navigation SET url = REPLACE(url, 'civicrm/tag', 'civicrm/tag/edi
 -- CRM-19815, CRM-19830 update references to check_number to reflect unique name
 UPDATE civicrm_uf_field SET field_name = 'contribution_check_number' WHERE field_name = 'check_number';
 UPDATE civicrm_mapping_field SET name = 'contribution_check_number' WHERE name = 'check_number';
+
+-- CRM-19715
+SELECT @closing_accounting_at := cov.value FROM civicrm_option_group cog
+  INNER JOIN civicrm_option_value cov 
```

Comment:
We should avoid using this join, even in upgrade scripts, it just encourages copy & paste.